lib/amqp.js

```diff
@@ -215,8 +215,9 @@ class Amqp {
         const payload = {
             error: encryptedError
         };
-
-        if (originalMessageContent && originalMessageContent !== '') {
+        if (headers.objectId) {
```

Comment:
It looks like your solution is not working. 
When sailor.js calls `amqp.sendError()` it passes not all incoming headers, but only subset of them (`taskId`, `execId`, `userId` and several others) `lib/sailor.js:219`. So `headers.objectId` will be always `undefined`.

Reply:
i understand the problem, thanks